blocks/cards/cards.js

```diff
@@ -4,6 +4,11 @@ export default function decorate(block) {
   /* change to ul, li */
   const ul = document.createElement('ul');
   [...block.children].forEach((row) => {
+    const numCards = block.children.length;
+    if (numCards % 2 === 0) {
+      // number of cards dividable by 2
+      block.classList.add('fifty-fifty');
+    }
```

Comment:
This is currently executed for each card (you don't see it in the DOM because the same class can only be added once). I'd move this 3 lines up and only have it execute once.

Reply:
fixed